go/live_updates_pub.go

```diff
@@ -0,0 +1,2 @@
+channel := client.Channels.Get("dogecoin:usd")
+channel.Publish(context.Background(), "rate", map[string]int{ "price": 2, "time": 543564332 })
```

Comment:
Please split this over two lines, and use variables in the message like [the JavaScript snippet](https://github.com/ably/homepage-snippets/blob/main/javascript/live-updates-pub.js)